docs/2_2_common_schema.adoc

```diff
@@ -1139,11 +1139,11 @@ Variable elements can uniformly represent variables of primitive (atomic) types,
 Variable elements representing array variables must contain a `Dimensions` element specifying the array dimensions.
 The `Dimensions` element contains a sequence of `Dimension` elements, each specifying the size of one dimension of the array:
 
-- If the `<<start>> attribute of the `Dimension` element is present, it defines a constant size for this dimension, namely the integer value of the <<start>> attribute.
+- If the <<start>> attribute of the `Dimension` element is present, it defines a constant `fmi3Uint32` size for this dimension.
 The <<variability>> of the dimension size is <<constant>> in this case.
 
 - If the <<valueReference>> attribute of the `Dimension` element is present, it defines the size of this dimension to be the value of the variable with the value reference given by the <<valueReference>> attribute.
-The referenced variable must be a variable of integer type, and must either be a constant (i.e. with <<variability>> = <<constant>>) or a <<structuralParameter,`structural parameter`>>(i.e. with <<causality>> = <<structuralParameter>>).
+The referenced variable must be a variable of `fmi3Uint32` type, and must either be a constant (i.e. with <<variability>> = <<constant>>) or a <<structuralParameter,`structural parameter`>>(i.e. with <<causality>> = <<structuralParameter>>).
```

Comment:
```suggestion
The referenced variable must be a variable of `UInt32` type, and must either be a constant (i.e. with <<variability>> = <<constant>>) or a <<structuralParameter,`structural parameter`>>(i.e. with <<causality>> = <<structuralParameter>>).
```
Better to use the XML schema element name, rather than the API type definition, I think.